source/settings/core.rst

```diff
@@ -4291,6 +4291,16 @@ See :ref:`dovecot_ssl_configuration`
 
 .. todo:: Indicate SSL setting
 
+.. _setting-ssl_cipher_suites:
+
+``ssl_cipher_suites``
+---------------------
+
+ - Default: OpenSSL version specific.
+
+.. versionadded:: v2.3.15
+
+The list of SSL cipher suites to use, in order of preference.
```

Comment:
Could have some kind of example values. and/or link to OpenSSL documentation about it.